package.json

```diff
@@ -12,18 +12,20 @@
   },
   "version": "1.6.4",
   "scripts": {
-    "build": "webpack -p --progress --mode production --config webpack.config.js",
-    "predeploy": "npm run build",
+    "build": "npx webpack -p --progress --mode production --config webpack.config.js",
```

Comment:
`webpack` should be installed locally. I don't we need `npx` here.

Reply:
Umm yes that was my fault, I usually use `npx` to execute local command. Let me fix it